backend/controllers/prisonerProfile/prisonerCellHistory.js

```diff
@@ -82,18 +75,7 @@ module.exports = ({ oauthApi, prisonApi, logError, page = 0 }) => async (req, re
             }
           })
         : [],
-      currentLocation: {
-        establishment: agencyData.find(agencyDetails => currentLocation.agencyId === agencyDetails.agencyId)
-          .description,
-        location: extractLocation(currentLocation.description, currentLocation.agencyId),
-        movedIn: currentLocation.assignmentDateTime && formatTimestampToDateTime(currentLocation.assignmentDateTime),
-        assignmentDateTime: moment(currentLocation.assignmentDateTime).format('YYYY-MM-DDTHH:mm:ss'),
-        assignmentEndDateTime: currentLocation.assignmentEndDateTime
```

Comment:
Again, checking for `cell.assignmentEndDateTime` but `currentLocation` is the cell which doesn't have an `assignmentEndDateTime`, so not needed.